ee/tabby-ui/components/user-avatar.tsx

```diff
@@ -12,29 +14,43 @@ import {
 } from '@/components/ui/avatar'
 import { Skeleton } from '@/components/ui/skeleton'
 
+const NOT_FOUND_ERROR = 'not_found'
+let fetchAvatar = true
```

Comment:
rename it to `shouldFetchAvatar` would be better

Reply:
updated